rust-toolchain.toml

```diff
@@ -1,5 +1,5 @@
 [toolchain]
-channel    = "1.78.0"
+channel    = "1.81.0"
```

Comment:
Is there any reason we upgraded to `1.8.0`?

Reply:
it is the same as parity uses, and it is stable. Do you have any concerns?

Reply:
Nope, just wondering! Thanks!